uint8[] classifiers
string name
